_data/sidebars/docs_sidebar.yml

```diff
@@ -241,9 +241,31 @@ entries:
       url: /tooling-rbf-shape.html
       output: web, pdf
 
-    - title: Micro-Manager
-      url: /tooling-micro-manager.html
+    - title: Micro Manager
+      url: /tooling-micro-manager-overview.html
       output: web, pdf
+      subfolders:
+
+      - title: Installation
+        url: /tooling-micro-manager-installation.html
+        output: web, pdf
+      
+      - title: Usage
+        url: /tooling-micro-manager-usage.html
+        output: web, pdf
+        subfolders:
+
+        - title: Code Changes
+          url: /tooling-micro-manager-usage-code-changes.html
+          output: web, pdf
+
+        - title: Configuration
+          url: /tooling-micro-manager-usage-configuration.html
+          output: web, pdf
+
+        - title: Running
+          url: /tooling-micro-manager-usage-running.html
+          output: web, pdf
```

Comment:
Do we really need this second hierarchy level `Usage`? Even large packages such as the OpenFOAM adapter don't have this.

Reply:
We do not have the second hierachy level anymore, but the Usage page was supposed to give the user an overview over what has to be done to adapt their code, do you think the other pages are expressive enough by themselves?

Reply:
It should be pretty clear from the order that the sections are instructions to be run one after the other. If in the future we get feedback about the section not being clear enough, we can introduce the Usage level again.